pkg/operator/ceph/csi/spec.go

```diff
@@ -584,6 +583,7 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 			return errors.Wrapf(err, "failed to start cephfs provisioner deployment: %+v", cephfsProvisionerDeployment)
 		}
 		k8sutil.AddRookVersionLabelToDeployment(cephfsProvisionerDeployment)
+		logger.Info("successfully started CSI CephFS driver")
```

Comment:
Shall we have a separate message for the volume plugin (line 555) and the provisioner here?
```suggestion
		logger.Info("successfully started CSI CephFS provisioner")
```

Reply:
@travisn as we are starting both provisioner and daemonset I left the logging as it was previously